### DISCUSSION TITLE
AMD OLED Power Optimization (Not sure if its related to g-helper)

### DISCUSSION THREAD

**Discussion by CLASSICCAR69** [seerge/g-helper#3373] [Q&A]
Hi, I have a 2024 G14. I recently updated to the stock amd drivers as the latest asus ones are from march 2024. After updating I found a new option in the AMD Adrenalin software called OLED Power Optimization. Upon Enabling it I found it gives the display a much richer contrast and would prefer to leave it on. However, after enabling it and going to a webpage or doing anything else the screen gets a white hue and washes out all of the color. I'm unsure what can be causing this and any help would be appreciated.

**Comment by gouthamk16** [comment]
Ran into this issue today. Found no fixes yet.

**Comment by seerge** [comment]
@gouthamk16 @CLASSICCAR69 hello, what specific setting are you talking about ? Can you post a screenshot ?

I also strongly advice to disable `Automatically Manage Color For Apps` toggle in windows, as it might affect color s a lot
<img width="2148" height="1236" alt="image" src="https://github.com/user-attachments/assets/0f92860a-bbaf-490b-bcc4-c6e9da7d3f1b" />

**Comment by ADVARZ** [comment]
I also have this issue.  Here's a screenshot of the specific setting 
<img width="1803" height="1082" alt="AMD power optimisation " src="https://github.com/user-attachments/assets/8a248157-7d91-42a9-9a22-3103a8f30bdd" />

**Reply by iSlidex** [reply]
set a lower brightness level on that same configuration screen, im using -75 right now with Oled power optimization.

**Comment by glebond701-source** [comment]
Same story, I tried to even install the official drivers but that didn't helped at all, it would be useful function because it helps battery life
